check/test_helpers.go

```diff
@@ -13,16 +13,17 @@ import (
 
 func newTestCheck() *HealthCheck {
 	config := HealthCheckConfig{
-		MessageLength:        100,
-		CheckInterval:        1 * time.Millisecond,
-		retryInterval:        1 * time.Millisecond,
-		CheckTimeout:         5 * time.Millisecond,
-		DataWaitInterval:     1 * time.Millisecond,
-		NoTopicCreation:      true,
-		topicName:            "health-check",
-		replicationTopicName: "replication-check",
-		brokerID:             1,
-		statusServerPort:     8000,
+		MessageLength:           100,
```

Comment:
I think that we should use this data structure to parameterise the retrier, for both number of retries and gap between them